hexrd/sampleOrientations/conversions.py

```diff
@@ -12,9 +12,7 @@
 
 @numba_njit_if_available(cache=True, nogil=True)
 def getPyramid(xyz):
-    x = xyz[0]
-    y = xyz[1]
-    z = xyz[2]
+    x, y, z = xyz
```

Comment:
Please include a check that `len(xyz)` is 3 and is an unpackable object. Otherwise, this will raise a `ValueError`.

Reply:
I assume it should raise an error anyway, but I'll make it a more meaningfully named error.

Reply:
[Fix?](https://github.com/HEXRD/hexrd/blob/0d1a11fb0e71b1f4d3a3c5a125dca15bc5d9367b/hexrd/sampleOrientations/conversions.py#L15-L18)